LostAtlasCAH.React/src/shared/components/GameSettings.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react";
+import Option from "./Option";
+
+class GameSettings extends Component {
+  render() {
+    return (
+      <div id="settings">
```

Comment:
So there's an idea called DRY (Don't Repeat Yourself) in coding, where you try to refactor duplicated code to make it more efficient and readable. I see a lot of similar things here's being repeated, is there any chance this could be rewritten as a method where you input: Title, SelectType, SelectSize, etc. ? Just an idea, This article explains what I mean a bit better: https://www.taniarascia.com/refactoring-dont-repeat-yourself/

Reply:
Definitely a good suggest, I'll see what I can figure out.